Source/ResourceLib.NetStandard/DeviceIndependentBitmap.cs

```diff
@@ -0,0 +1,179 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Runtime.InteropServices;
+using System.ComponentModel;
+using System.IO;
+
+namespace Vestris.ResourceLib
+{
+    /// <summary>
+    /// A device-independent image consists of a BITMAPINFOHEADER where
+    /// bmWidth is the width of the image andbmHeight is double the height 
+    /// of the image, followed by the bitmap color table, followed by the image
+    /// pixels, followed by the mask pixels.
+    /// </summary>
+    public class DeviceIndependentBitmap
```

Comment:
Why did you copy into a new file?

Reply:
I copied this into a new file because I needed to remove a lot of code from this file compared to the full-framework version. I didn't think of `#if NETSTANDARD` until after I already completed this part. Will go back and fix.